test/SwitchTransition-test.js

```diff
@@ -106,6 +106,9 @@ describe('SwitchTransition', () => {
     act(() => {
       jest.runAllTimers();
     });
+    act(() => {
+      jest.runAllTimers();
+    });
```

Comment:
Why is this call added?

Reply:
Let me check. I think it was due to previous tests relying on implementation details related to React scheduling timings. Will check what specifically was happening.